products/BellHybrid/apps/application-bell-relaxation/presenter/RelaxationRunningProgressPresenter.cpp

```diff
@@ -8,6 +8,15 @@
 #include <common/models/TimeModel.hpp>
 #include <gsl/assert>
 
+namespace
+{
+    bool songLengthEqualsToSelectPeriod(std::chrono::minutes period, std::chrono::seconds songLength)
```

Comment:
```suggestion
    bool songLengthEqualsToSelectedPeriod(std::chrono::minutes period, std::chrono::seconds songLength)
```

You can also pass the values by const ref to avoid copying

Reply:
There is no difference, I've already checked this :P 

https://stackoverflow.com/questions/37918168/pass-stdchronoduration-by-value-or-by-reference-to-const